gwswlib/scripts.py

```diff
@@ -46,6 +56,21 @@ def run_import_export(
     else:
         raise OptionException("no available export type %r is selected" % export_type)
 
+    if log_relpath is not None:
+        logger.info("Log file with warnings and errors is created %r", log_relpath)
+
+    logger.info("Exchange of GWSW-hydx finished")
+
+    return "method is finished"
```

Comment:
Deze return value is alleen voor het testen? Dan zou ik er zoiets achter zetten: `# Return value only for testing`.
Even om duidelijk te maken dat er (afgezien van de tests) niets afhankelijk is van de specifieke return value.